examples/motoko_examples/factorial/dfx.json

```diff
@@ -1,14 +1,28 @@
 {
     "canisters": {
-        "factorial": {
+        "azle": {
             "type": "custom",
-            "build": "npx azle factorial",
-            "root": "src",
-            "ts": "src/factorial.ts",
-            "candid": "src/factorial.did",
-            "wasm": "target/wasm32-unknown-unknown/release/factorial.wasm",
+            "build": "npx azle azle",
+            "root": "canisters/azle",
+            "ts": "canisters/azle/index.ts",
+            "candid": "canisters/azle/index.did",
+            "wasm": "target/wasm32-unknown-unknown/release/azle.wasm",
             "declarations": {
-                "output": "test/dfx_generated/factorial"
+                "output": "dfx_generated/azle"
+            }
+        },
+        "motoko": {
+            "main": "canisters/motoko/main.mo",
```

Comment:
Same as in the other pull request, let's make the dfx.json consistent with the others